docs/package.json

```diff
@@ -4,11 +4,11 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "build": "docusaurus build",
+    "start": "cp -f ../CHANGELOG.md src/pages/changelog.md && docusaurus start",
```

Comment:
Cleaner would be probably to integrate the existing CHANGELOG.md with some JSX. But this suits the bill just fine.

Reply:
Not possible since it's out-of-tree. I tried this first. Second attempt was a repository-internal symlink, but that's broken in the webpack version that docusaurus currently requires.